queries/pascal/injections.scm

```diff
@@ -0,0 +1,2 @@
+(comment) @comment
+(asmBody) @asm
```

Comment:
we don't have support for asm yet, you can comment this to add asm when it's added.

Reply:
I added a comment and commented out the part of the code. I'm assuming that's what you meant. Let me know.